src/devices/MAX31856/samples/MAX31856.Sample.cs

```diff
@@ -17,13 +18,15 @@
 using MAX31856 sensor = new(device, ThermocoupleType.K);
 while (true)
 {
-    // read temperature
-    var temp = sensor.GetTemperature().DegreesFahrenheit;
-    temp = Math.Round(temp, 7); // round temp output to seven significant figures from resolution in Technical Documentation
-    // read cold junction temperature of device
-    var tempCJ = sensor.GetCJTemperature().DegreesFahrenheit;
-    tempCJ = Math.Round(tempCJ, 2); // round temp output to two significant figures from resolution in Technical Documentation
-    Console.WriteLine($"Temp: {temp} ColdJunctionTemp: {tempCJ} ");
+    // Reads temperature if the device is not reading properly
+    sensor.TryGetTemperature(out Temperature temperature);
```

Comment:
with the TryReadTemperature pattern, it should then be like:

```csharp
var tempColdJunction = sensor.GetCJTemperature();
if (sensor.TryGetTemperature(out Temperature temperature))
{
  Console.WriteLine($"Temperature: {temperature.DegreesFahrenheit:0.0000000} °F, Cold Junction: {tempColdJunction.DegreesFahrenheit:0.00} °F");
}
else
{
  Console.WriteLine($"Error reading temperature, Cold Junction temperature: {tempColdJunction.DegreesFahrenheit:0.00}");
}
```

Reply:
This is great thank you so much.